docs/scos/dev/guidelines/testing-guidelines/executing-tests/test-the-glue-api.md

```diff
@@ -0,0 +1,395 @@
+     *
+     * @param \PyzTest\Glue\Wishlists\WishlistsApiTester $I
+     *
+     * @return void
+     */
+    public function requestWishlistByUuid(WishlistsApiTester $I): void
+    {
+        // Arrange
+        $wishlistUuid = $this->fixtures->getWishlistTransfer()->getUuid();
+        $oauthResponseTransfer = $I->haveAuthorizationToGlue($this->fixtures->getCustomerTransfer());
+        $I->amBearerAuthenticated($oauthResponseTransfer->getAccessToken());
+        $url = $I->buildWishlistUrl($wishlistUuid);
+
+        // Act
+        $I->sendGET($url);
+
+        // Assert
+        $I->seeResponseCodeIs(HttpCode::OK);
+        $I->seeResponseIsJson();
+        $I->seeResponseMatchesOpenApiSchema();
+
+        $I->amSure('returned resource is of correct type')
+            ->whenI()
+            ->seeResponseDataContainsSingleResourceOfType(WishlistsRestApiConfig::RESOURCE_WISHLISTS);
+
+        $I->amSure('returned resource has correct id')
+            ->whenI()
+            ->seeSingleResourceIdEqualTo($wishlistUuid);
+
+        $I->amSure('returned resource has correct self-link')
+            ->whenI()
+            ->seeSingleResourceHasSelfLink($url);
+    }
+
+    /**
+     * @depends loadFixtures
+     *
+     * @param \PyzTest\Glue\Wishlists\WishlistsApiTester $I
+     *
+     * @return void
+     */
+    public function requestWishlists(WishlistsApiTester $I): void
+    {
+        $oauthResponseTransfer = $I->haveAuthorizationToGlue($this->fixtures->getCustomerTransfer());
+        $I->amBearerAuthenticated($oauthResponseTransfer->getAccessToken());
+        $wishlistUuid = $this->fixtures->getWishlistTransfer()->getUuid();
+
+        // Act
+        $I->sendGET($I->buildWishlistsUrl());
+
+        // Assert
+        $I->seeResponseCodeIs(HttpCode::OK);
+        $I->seeResponseIsJson();
+        $I->seeResponseMatchesOpenApiSchema();
+
+        $I->amSure('Response data contains resource collection')
+            ->whenI()
+            ->seeResponseDataContainsResourceCollectionOfType(WishlistsRestApiConfig::RESOURCE_WISHLISTS);
+
+        $I->amSure('Resource collection has resource')
+            ->whenI()
+            ->seeResourceCollectionHasResourceWithId($wishlistUuid);
+
+        $I->amSure('Resource has correct self-link')
+            ->whenI()
+            ->seeResourceByIdHasSelfLink($wishlistUuid, $I->buildWishlistUrl($wishlistUuid));
+    }
+}
+```
+
+### 3. Run the test
+
+#### 1. Build the fixtures
+
+Run the following command to generate fixtures:
+
+```bash
+docker/sdk testing codecept fixtures
+```
+
+#### 2. Synchronize data
+
+Run the following command to synchronize the data to storage/search:
+
+```bash
+docker/sdk testing console queue:worker:start --stop-when-empty
+```
+
+#### 3. Execute tests
+
+Run the following command to execute the tests:
+
+```bash
+docker/sdk testing codecept run -c tests/PyzTest/Glue/Wishlists
+```
+
+Once the testing process is complete, you get the result of each individual test.
```

Comment:
* The result of each individual test will be displayed once the testing process is complete.